images/kubekins-e2e/Dockerfile

```diff
@@ -57,6 +57,9 @@ ENV GO_TARBALL "go${GO_VERSION}.linux-amd64.tar.gz"
 RUN wget -q "https://storage.googleapis.com/golang/${GO_TARBALL}" && \
     tar xzf "${GO_TARBALL}" -C /usr/local && \
     rm "${GO_TARBALL}"
+# install dep and ko since they are common dependencies for go projects related to k8s
+RUN go get github.com/golang/dep/cmd/dep && \
+    go get github.com/google/go-containerregistry/cmd/ko
```

Comment:
Why not do this in the job? This shouldn't take long and if we do it in the image we now potentially have a versioning problem for these tools.

Reply:
Installing it in the job is also an option. I have an internal change for our team in this approach. 

The change I am trying to make is this two tools, at least `dep`, is commonly used across multiple k8s projects. And it would be nice for it to be part of the toolchain so that each test job don't need to install it everytime.

Reply:
The problem is that dep like most tools changed a bit between versions, putting it in a shared image makes it tricky to avoid breaking everyone suddenly when we upgrade the image. That's why EG golint is not in this image.

Reply:
Er, in general putting tools like this in _an_ image is a great idea, it's faster at runtime and allows controlling the versions by controlling the image version. The problem is that consumers are not maintaining this image themselves, it exists primarily for Kubernetes/Kubernetes and other projects just happen to use it. If your project can, it should switch to using your own image(s) that you update at your own pace.

Reply:
I feel like you don't have to depend on dep in CI, why not commit all vendor code and always keep your branch up-to-date? We (test-infra repo) uses dep but didn't require dep in CI afaik (we use the same kubekins image)

Reply:
OK thanks for the review! I will close this PR for now.